### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Polokghosh53**
Hello my name is Polok Ghosh and I am a participant from GSSOC '21
and I thought I shall improve the readme for the users interacting for the use in their personal computer and also for me to get started with Open Source contribution.

### CODE REVIEW

**Comment by yvkrishna** [comment]
> Hello my name is Polok Ghosh and I am a participant from GSSOC '21
> and I thought I shall improve the readme for the users interacting for the use in their personal computer and also for me to get started with Open Source contribution.

Hello polok ghosh.
I have looked into your pr. Its good. But managing pr's is both the admin and mentors part. our team will be evaluating  the pr's.  This may take some time because the project period has just started.

**Comment by Polokghosh53** [comment]
No problem @yvkrishna . Your confirmation is enough for me. Please take your time. Thank you.

**Review by chebroluharika — APPROVED** [review]
LGTM

**Comment by Polokghosh53** [comment]
Thank you

**Comment by Polokghosh53** [comment]
Thank you

**Comment by Polokghosh53** [comment]
When will it be merged?

File: README.md [hunk 1]
```diff
@@ -16,22 +16,51 @@ Flow2Ml</p>
...
+<li>sklearn</li>
+
+```
+$ pip install scikit-learn
+```
+    
+<li>numpy</li>
+
+```
+$ pip install scikit-learn
```
Comment (chebroluharika) [inline README.md:54]: Duplicate of 48. Please change this to `pip install numpy`

Comment (chebroluharika) [reply README.md:54]: I think now this individual dependencies installation won't make sense as those are getting installed using requirements file. Installing through requirements file is a new enhancement made to repo.
Hence, please revert those changes.

**Comment by Polokghosh53** [comment]
During resolving the conflicts, I have to do correction to the duplicate of 48. But it is saying that I have solve 2 conflicts which is the other one?

**Comment by chebroluharika** [comment]
You have 2 conflicts
One is at dependencies section and other is at Contributing section

**Comment by Polokghosh53** [comment]
So, at the contributing section at which line does the change happen and what exactly do I have to mention?

**Comment by chebroluharika** [comment]
@Polokghosh53 For resolving conflicts, you need to keep an eye with code whatever is present in main branch. Then you need not to change code of main branch unless you have your own changes for the same set of lines. That's how we should resolve conflicts. Anyways for now I resolved those conflicts for this PR..

**Comment by chebroluharika** [comment]
@Polokghosh53 , @yvkrishna  I am changing the level of this PR as these are minor Documentation changes.

**Review by chebroluharika — APPROVED** [review]

**Pull request merged**